tests/integration/test_sns.py

```diff
@@ -607,6 +610,42 @@ def test_create_duplicate_topic_with_different_tags(self):
         # clean up
         self.sns_client.delete_topic(TopicArn=topic_arn)
 
+    def test_publish_by_path_parameters(self):
+        topic_name = 'topic-{}'.format(short_uid())
+        queue_name = 'queue-{}'.format(short_uid())
+
+        message = 'test message {}'.format(short_uid())
+        topic_arn = self.sns_client.create_topic(Name=topic_name)['TopicArn']
+
+        base_url = '{}://{}:{}'.format(get_service_protocol(), config.LOCALSTACK_HOSTNAME, config.PORT_SNS)
+        path = 'Action=Publish&Version=2010-03-31&TopicArn={}&Message={}'.format(topic_arn, message)
+
+        queue_url = self.sqs_client.create_queue(QueueName=queue_name)['QueueUrl']
+        queue_arn = aws_stack.sqs_queue_arn(queue_name)
+
+        self.sns_client.subscribe(TopicArn=topic_arn, Protocol='sqs', Endpoint=queue_arn)
+
+        r = requests.post(
+            url='{}/?{}'.format(base_url, path),
+            headers={
+                'Authorization': 'AWS4-HMAC-SHA256 Credential=test-id/20200609/us-east-1/sns/aws4_request, '
```

Comment:
nit: would be great if we could use `aws_stack.mock_aws_request_headers('sns')` here, instead of hardcoding the header value. 

Note: This may require a minor adjustment in `mock_aws_request_headers(..)`, something like;:
```
    elif service in ['sns', 'sqs']:
        ctype = APPLICATION_X_WWW_FORM_URLENCODED
```